packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx

```diff
@@ -110,8 +119,32 @@ export const CommentsSection: React.FC<CommentsSectionProps> = ({ disabled, vide
     return () => clearTimeout(timeout)
   })
 
+  // increase number of comments when user scrolls to the end of page
```

Comment:
Unfortunately, this won't work well on smaller viewports, the comment section isn't last section on mobile: 
![image](https://user-images.githubusercontent.com/51168865/170662599-a154327e-95f7-45de-9a58-4096388c6fc5.png)
We would have to scroll down to CTA buttons and then scroll up to see new comments.